install/ansible/install.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
```

Comment:
Alpine doesn't have bash, so I modified the redirect instead of adding another dependency.